docs/storage.md

```diff
@@ -68,6 +68,90 @@ Pulp operator will automatically configure Pulp `settings.py` with the provided
 !!! info
     Only one type of Object Storage should be provided. Trying to declare both will fail operator execution.
 
+### Configuring Azure Blob
+
+#### Prerequisites
+* To configure Pulp with Azure Blob as a storage backend, the first thing to do is create an [Azure Storage Blob Container](https://docs.microsoft.com/en-us/azure/storage/blobs/quickstart-storage-explorer) to store the objects.
+* After configuring a `Blob Container`, take a note of the [Azure storage account](https://docs.microsoft.com/en-us/azure/storage/common/storage-account-get-info?toc=%2Fazure%2Fstorage%2Fblobs%2Ftoc.json&tabs=portal)
+
+After performing all the prerequisites, create a `Secret` with them:
+```
+$ PULP_NAMESPACE='my-pulp-namespace'
+$ AZURE_ACCOUNT_NAME='my-azure-account-name'
+$ AZURE_ACCOUNT_KEY='my-azure-account-key'
+$ AZURE_CONTAINER='pulp-test'
+$ AZURE_CONTAINER_PATH='pulp3'
+$ AZURE_CONNECTION_STRING='my-azure-connection-string'
```

Comment:
the connection string is not required, we use it because it is the way to make [azurite](https://github.com/Azure/Azurite) work

Reply:
Should I remove it from doc or would be better to put a comment saying that it is optional?

Reply:
the latter